tardis/utilities/executors/sshexecutor.py

```diff
@@ -180,3 +181,17 @@ async def run_command(self, command, stdin_input=None):
                     stderr=response.stderr,
                     exit_code=response.exit_status,
                 )
+
+
+@enable_yaml_load("!DupingSSHExecutor")
+@yaml_tag(eager=True)
+class DupingSSHExecutor(SSHExecutor):
+    def __init__(self, **parameters):
+        self._wrapper_script = parameters.pop("wrapper", "/bin/bash")
+        super().__init__(**parameters)
```

Comment:
Bit of a nit and not sure what's the convention in the rest of Tardis. This should be semantically equivalent to having an actual parameter `wrapper`, which IMO gets the intention across better.
```suggestion
    def __init__(self, *, wrapper = "/bin/bash", **parameters):
        self._wrapper_script = wrapper
        super().__init__(**parameters)
```

Reply:
I think that is the better way. Thanks for your suggestion.